theseus/embodied/kinematics/robot_model.py

```diff
@@ -27,12 +31,12 @@ class IdentityModel(RobotModel):
     def __init__(self):
         super().__init__()
 
-    def forward_kinematics(self, robot_pose: LieGroup) -> torch.Tensor:
+    def forward_kinematics(self, robot_pose: RobotModelInput) -> RobotModelOuput:
         if isinstance(robot_pose, SE2):
-            return robot_pose.translation.data.view(-1, 2, 1)
+            return robot_pose.translation
         if isinstance(robot_pose, Point2) or isinstance(robot_pose, Vector):
             assert robot_pose.dof() == 2
-            return robot_pose.data.view(-1, 2, 1)
+            return robot_pose
```

Comment:
The output should be a LieGroup, but robot_pose is a `Point2` or a `Vector` here?

Reply:
`Point2` is derived from `Vector` which is derived from `LieGroup` :)

Reply:
Ahh I wasn't aware of that. Thanks!